style.css

```diff
@@ -535,3 +545,314 @@ input {
   background: white;
   color: #5e6c84;
 }
+
+@media only screen and (min-width: 992px) {
+  body {
+    background: #e5e5e5;
+    max-width: 1100px;
+  }
+
+  .navbar {
+    width: 1350px;
+    padding: 0;
+    display: flex;
+    padding-right: 155px;
+    padding-left: 155px;
+    padding-top: 25px;
+    padding-bottom: 25px;
+  }
+
+  .navbar ul,
+  .logo {
+    margin: 0;
+  }
+
+  .navbar ul {
+    display: flex;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .navbar ul a {
+    padding-right: 50px;
+    display: flex;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+  }
+
+  .hero {
+    width: 1350px;
+    height: 700px;
+    background: rgb(255, 255, 255) url("./images/Header bg.svg") no-repeat;
+    background-size: contain;
+    padding: 24px;
+  }
+
+  .title {
+    margin-top: 80px;
+    margin-bottom: 0;
+  }
+
+  .title h1,
+  .title h2 {
+    font-style: normal;
+    font-weight: bold;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    width: 800px;
+    margin: 0 auto;
+  }
+
+  .description p {
+    width: 800px;
+    height: 98px;
+  }
+
+  .connect {
+    width: 150px;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+    margin-top: 0;
+    text-align: left;
+    margin-left: 0;
+  }
+
+  .social {
+    margin-left: 250px;
+    display: block;
+  }
+
+  .social img {
+    width: 20px;
+    height: 20px;
+  }
+
+  .about-wrapper {
+    width: 1350px;
+    height: 668px;
+    padding: 90px;
+  }
+
+  .headline-about {
+    display: flex;
+  }
+
+  .about-flex {
+    overflow: hidden;
+    padding: 30px;
+  }
+
+  .social-about {
+    margin: 0;
+  }
+
+  .languages {
+    margin: 0;
+  }
+
+  .language-wrapper {
+    padding: 30px;
+  }
+
+  .language-wrapper .tech {
+    width: 582px;
+  }
+
+  .items-container {
+    display: flex;
+    padding: 0;
+  }
+
+  .items {
+    display: flex;
+    flex-direction: column;
+    width: 122px;
+    height: 120px;
+    margin: 0 12px;
+    padding: 10px;
+    margin-left: 0;
+  }
+
+  .items > span {
+    margin-top: 16px;
+  }
+
+  .about {
+    margin: 0;
+  }
+
+  .about h2,
+  .about span {
+    margin-left: 0;
+  }
+
+  .about-desc {
+    width: 447px;
+    height: 114px;
+    margin: 0;
+    margin-bottom: 12px;
+    margin-top: 15px;
+  }
+
+  .form-section {
+    min-width: 1350px;
```

Comment:
- [ ] I know that you are following the template by using the same width but for this project as we are adapting it to different kinds of screens. To mean it would be better for all with of containers to take `100%` instead of a fixed with. Kindly use `min-width: 100%` instead of `min-width: 1350px;` in the `form-section` class.This will adapt our design to all screens greater than 992px.